### ISSUE TITLE
Update broker to use parameterized IAM group policy

### ISSUE THREAD

**Issue by markdboyd** [cloud-gov/aws-broker#252]
In order to reduce the number of IAM users created/managed by the broker, the broker should have a parameterized group policy and add the user/role to that group instead of the broker making an individual policy for each instance.

Recommendation came out of security consulting from AWS. See [the meeting notes](https://docs.google.com/document/d/1BfWD3P7mzeP2sGE9HSJloMX4dS4tq02A7FQEbgMjhJo/edit)

## POC

Ask @rbogle for further details if necessary

**Comment by pburkholder** [comment]
Should this be included with broader AWS security recommendations issue?

**Comment by markdboyd** [comment]
@pburkholder Yes, it is tracked in that epic: https://github.com/cloud-gov/product/issues/2272
